pkg/apis/core/helper/helpers.go

```diff
@@ -495,3 +496,17 @@ func IsMultiZonalSeed(seed *core.Seed) bool {
 func IsHASeedConfigured(seed *core.Seed) bool {
 	return metav1.HasLabel(seed.ObjectMeta, v1beta1constants.LabelSeedMultiZonal) || seed.Spec.HighAvailability != nil
 }
+
+// ValidateBooleanValue validates the value of label `v1beta1constants.LabelSeedMultiZonal`.
+func ValidateBooleanValue(val string, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	if len(val) == 0 {
+		return allErrs
+	}
```

Comment:
```suggestion
```

I thing accepting an empty value is not expected for a generic function like this. Let's remove it here and check if `val != ""` before invoking this (see corresponding comment below).

Reply:
I never wanted to make it a generic method to start with. This method to start with was very specific to multi-zone label. Fine i will also remove this empty check. To be honest this generic method is now just validating a boolean value and IMHO it is an overkill to have a utility method for it.

Reply:
After offline discussion with Tim we agreed to revert this generalization of this specific use case and only have it for checking the multi-zonal seed label.

Reply:
As discussed, fine for me to revert it.

Reply:
done